package.json

```
@@ -30,7 +30,7 @@
     "node": ">=10"
   },
   "dependencies": {
-    "@erc725/erc725.js": "^0.21.0",
+    "@erc725/erc725.js": "^0.21.2",
     "@ethersproject/abi": "^5.7.0",
     "@ethersproject/experimental": "^5.7.0",
     "@lukso/lsp-smart-contracts": "^0.12.0",
```

@richtera @CallumGrindle 

Please use this patch version, so that the lsp-factory deploys the patched version of the contracts without the bugs

```suggestion
    "@lukso/lsp-smart-contracts": "^0.12.1",
```

---

Being done in #235